.github/workflows/cmake.yml

```
@@ -172,8 +248,8 @@ jobs:
       - name: Set environment variables
         run: |
           echo "COLUMNS=79" >> ${GITHUB_ENV}
-          echo "LD_LIBRARY_PATH=${PWD}/install/lib" >> ${GITHUB_ENV}
-          echo "PKG_CONFIG_PATH=${PWD}/install/lib/pkgconfig" >> ${GITHUB_ENV}
+          echo "LD_LIBRARY_PATH=${PWD}/install/lib:${{ env.LD_LIBRARY_PATH }}" >> ${GITHUB_ENV}
```

In general, we should use `${XXX}` instead of `${{ env.XXX }}` for readability and simplicity:

```suggestion
          echo "LD_LIBRARY_PATH=${PWD}/install/lib:${LD_LIBRARY_PATH}" >> ${GITHUB_ENV}
```